### PR TITLE
Test serialize MultipleOrderResponse and run cargo fmt

### PR DESCRIPTION
**Pull request by matteosantama**
I've raised several different issues, but I didn't want to cram them all into one PR. This PR takes steps to circumvent the failures in https://github.com/dpeachpeach/kalshi-rust/issues/7, fixes the serialization of `MultipleOrderResponse` as described in https://github.com/dpeachpeach/kalshi-rust/issues/6, and runs `cargo fmt` on the codebase (sorry, it's a habit of mine).

### CODE REVIEW

**Comment by dpeachpeach** [comment]
Great, thank you so much!

I'll review this either tonight or tomorrow.

File: kalshi/src/portfolio.rs [hunk 1]
```diff
@@ -1057,49 +1068,70 @@ pub enum OrderType {
...
+
+    #[test]
+    fn test_serialize_multiple_order_response() -> serde_json::Result<()> {
+        let json = r#"{"orders":[],"cursor":""}"#;
+        let result = serde_json::from_str::<MultipleOrderResponse>(json)?;
+        assert!(result.orders.is_empty());
+        assert!(result.cursor.is_none());
+        Ok(())
+    }
+}
```
Comment (matteosantama) [inline kalshi/src/portfolio.rs:1137]: @dpeachpeach Sorry, the diff is hard to parse because of all the formatting changes. This is the critical part of the PR -- the rest should just be whitespace manipulation.

File: kalshi/Cargo.toml [hunk 2]
```diff
@@ -12,12 +12,18 @@ repository = "https://github.com/dpeachpeach/kalshi-rust"
...
 [dependencies]
 reqwest = { version = "0.11", features = ["json"] }
 tokio = { version = "1", features = ["full"] }
 serde = { version = "1.0", features = ["derive"]}
-uuid = { version = "1.5.0", features = ["v4", "fast-rng"]}
\ No newline at end of file
+uuid = { version = "1.5.0", features = ["v4", "fast-rng"]}
+
+[dev-dependencies]
+serde_json = "1.0.111"
```
Comment (matteosantama) [inline kalshi/Cargo.toml:29]: The other important part of the PR

**Review by dpeachpeach — APPROVED** [review]
Everything looks good

**Comment by matteosantama** [comment]
Awesome, I don't have write access to the repository so I'll leave it for you to hit merge!

**Comment by dpeachpeach** [comment]
Sure thing, I'll merge sometime this weekend.
